cmd/shortener/api/api.go

```diff
@@ -0,0 +1,79 @@
+package api
+
+import (
+	"fmt"
+	"github.com/KorsakovPV/shortener/cmd/shortener/config"
+	"github.com/go-chi/chi/v5"
+	"github.com/google/uuid"
+	"io"
+	"log"
+	"net/http"
+)
+
+type abstractStorage interface {
```

Comment:
We should use `interface` between packages. But we can use `struct` for one package.

Reply:
ok